pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -3211,6 +3230,59 @@ func TestListPlugins(t *testing.T) {
 	}
 }
 
+func TestClose(t *testing.T) {
+	tests := []struct {
+		name    string
+		plugins *config.Plugins
+		wantErr string
+	}{
+		{
+			name: "close doesn't return error",
+			plugins: &config.Plugins{
+				MultiPoint: config.PluginSet{
+					Enabled: []config.Plugin{
+						{Name: testPlugin, Weight: 5},
+					},
+				},
+			},
+		},
+		{
+			name: "close returns error",
+			plugins: &config.Plugins{
+				MultiPoint: config.PluginSet{
+					Enabled: []config.Plugin{
+						{Name: testPlugin, Weight: 5},
+						{Name: testErrorPlugin},
+					},
+				},
+			},
+			wantErr: "close err",
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			_, ctx := ktesting.NewTestContext(t)
+			ctx, cancel := context.WithCancel(ctx)
+			defer cancel()
+			fw, err := NewFramework(ctx, registry, &config.KubeSchedulerProfile{Plugins: tc.plugins})
+			if err != nil {
+				t.Fatalf("Unexpected error during calling NewFramework, got %v", err)
+			}
+			closeErr := fw.Close()
+			if closeErr != nil {
+				if tc.wantErr == "" || !strings.Contains(closeErr.Error(), tc.wantErr) {
```

Comment:
We can just do this, right? (maybe you have to modify wantErr in a second case though.

```suggestion
				if tc.wantErr == "" || closeErr.Error() != tc.wantErr {
```